### DISCUSSION TITLE
Splitting book collections into separate books

### DISCUSSION THREAD

**Discussion by Mezhona** [audiamus/AaxAudioConverter#169] [Ideas]
I have several books that are "Name of book (Book 1 - 9)", would it be possible to split them into separate books?
<img width="234" alt="image" src="https://user-images.githubusercontent.com/31044777/226404873-cf5097de-9d9c-413f-b9ce-cc9ce012be86.png">

**Comment by audiamus** [comment]
There is no direct support for this. Nonetheless, you could split by chapter first. Especially, if the chapter meta data of this collection contain explicit chapter titles, aka "Named chapters". Afterwards you could use an external tool like Mp3tag to further tailor meta titles and names of the chapter files.
